hack/.golint_failures

```diff
@@ -130,6 +130,7 @@ pkg/controller/volume/events
 pkg/controller/volume/expand
 pkg/controller/volume/persistentvolume
 pkg/controller/volume/persistentvolume/options
+pkg/controller/volume/pvprotection
```

Comment:
I'd appreciate if the PR could be without this line. I guess it complains about type/function names and a little rename would resolve it.

Reply:
thanks,done